cdapython/fetch.py

```diff
@@ -2269,61 +2227,43 @@ def fetch_rows(
         # Consolidate provenance information if present.
 
         if provenance == True:
-            
-            if table == 'somatic_mutation':
```

Comment:
this bit needs to be put back (just rename the table). there's no `mutation_identifier` table; we're drawing identifier and provenance data from `subject_identifier` and `subject_data_source`, resp., so the special handling needs to be retained. this may still result in an unhandled error regarding subject identifier info returned via `mutation` queries, i'm not completely sure -- testing will tell, but we definitely at least need to retain this bit.

Reply:
Yes you're right. I just saw this while looking at the rest.